assignments/resume-bootstrap/resume.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+    <title>Reem resume</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="bootstrap.min.css">
+    <link rel="stylesheet" href="resume.css">
+</head>
+
+<body>
+    <div class="container-fluid">
+        <div class="row">
+            <header class="header">
+                <div class="col-xs-12 col-md-5 col-sm-5" id="headerleft">
```

Comment:
Notice the redundant use of header in the id. We already know that this is part of the header.